rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/scan/ast/JavaTypeReferenceModel.java

```diff
@@ -18,8 +18,21 @@
 
     public static final String TYPE = "JavaTypeReference";
     public static final String REFERENCE_TYPE = "referenceType";
+    public static final String RESOLVED_SOURCE_SNIPPIT = "resolvedSourceSnippit";
 
     /**
+     * Gets the snippit referenced by this {@link FileLocationModel}.
+     */
+    @Property(RESOLVED_SOURCE_SNIPPIT)
+    void setResolvedSourceSnippit(String source);
```

Comment:
@mbriskar After looking over the way we use this value more, I don't think it is a big deal to change it to your approach at all. I'm ok with it. :)